examples/offloader.js

```diff
@@ -31,53 +33,54 @@ if (fs.existsSync('polldata.json')) {
 
 client.logOn(logOnOptions);
 
-client.on('loggedOn', function() {
-	console.log("Logged into Steam");
+client.on('loggedOn', function () {
+	console.log('Logged into Steam');
```

Comment:
@DoctorMcKay I personally prefer to have one types of quotes, as it makes codebase more consistent.
Anyway, we should ask the others.